insights/specs/default.py

```diff
@@ -380,13 +380,16 @@ def httpd_cmd(broker):
     ls_ocp_cni_openshift_sdn = simple_command("/bin/ls -l /var/lib/cni/networks/openshift-sdn")
     ls_origin_local_volumes_pods = simple_command("/bin/ls -l /var/lib/origin/openshift.local.volumes/pods")
     ls_osroot = simple_command("/bin/ls -lan /")
-    ls_run_systemd_generator = simple_command("/bin/ls -lan /run/systemd/generator")
+    ls_run_systemd = simple_command("/bin/ls -lanRL /run/systemd")
```

Comment:
@jobselko you will also need to make the corresponding change the spec name in [insights_archive.py](https://github.com/RedHatInsights/insights-core/blob/0f084dfc6b6f9785194ae6a02cee20ccd1e6ca79/insights/specs/insights_archive.py#L125)